lndr-backend/src/Lndr/Db/VerifiedCredits.hs

```diff
@@ -117,7 +121,7 @@ twoPartyBalance addr counterparty ucac conn = do
     return . floor $ balance
 
 
-twoPartyNonce :: Address -> Address -> Connection -> IO Nonce
-twoPartyNonce addr counterparty conn = do
+twoPartyNonce :: Address -> Address -> Integer -> Connection -> IO Nonce
```

Comment:
I don't think this makes sense. If you want increment a twoPartyNonce, why don't you do it with the return value of this function? This db access function should be kept as simple as possible. I think this `Integer` parameter should be removed and the function should be reverted to its old form.